### ISSUE TITLE
feat: add Wizard to/in button label to be parameter

### ISSUE THREAD

**Issue by tomokusaba** [microsoft/fluentui-blazor#1285]
# 🙋 Feature Request
I want the button label of the wizard to be a parameter.
It's understandable that you can do anything with a ButtonTemplate.
But all you want to do is change the label of the button from English to Japanese.
ButtenTemplate is just too hard for what you want to do.

## 🤔 Expected Behavior
Allows you to specify the labels of the Previous, Next, and Done buttons with parameters.

## 😯 Current Behavior
You have no choice but to create a button with a ButtonTemplate.

**Comment by dvoituron** [comment]
As describe on this page, you have static properties to do that: https://www.fluentui-blazor.net/Wizard#customized

> You can customize button labels using the **ButtonTemplate** or by modifying the static properties **FluentWizard**.**LabelButtonPrevious** / **LabelButtonNext** / **LabelButtonDone**.

If you want to configure in more detail (for example, different labels on different pages), you need to use the `ButtonTemplate` section.

```razor
<ButtonTemplate>
    @{
        var index = context;
        var lastStepIndex = 3;

        @if (index > 0)
        {
            <FluentButton OnClick="@(() => Value -= 1)">Previous</FluentButton>
        }

        @if (index < lastStepIndex)
        {
            <FluentButton OnClick="@(() => Value += 1)" Appearance="Appearance.Accent">Next</FluentButton>
        }
    }
</ButtonTemplate>
```

**Comment by tomokusaba** [comment]
> modifying the static properties FluentWizard.LabelButtonPrevious / LabelButtonNext / LabelButtonDone.

Do you have a sample somewhere?

**Comment by dvoituron** [comment]
> Do you have a sample somewhere?

To do what? To assign a static variable ?????

```csharp
FluentWizard.LabelButtonNext = ">";
FluentWizard.LabelButtonPrevious = "<";
FluentWizard.LabelButtonDone = "OK";
```

**Comment by tomokusaba** [comment]
Since it affects the entire application, there is the question of where to write it.
For example, I thought that if there was a guideline that it should be written in Program.cs, it would be better to have it in the document.
For example, I did the following:

Program.cs
```csharp
using BlazorApp26;
using Microsoft.AspNetCore.Components.Web;
using Microsoft.AspNetCore.Components.WebAssembly.Hosting;
using Microsoft.FluentUI.AspNetCore.Components;

var builder = WebAssemblyHostBuilder.CreateDefault(args);
builder.RootComponents.Add<App>("#app");
builder.RootComponents.Add<HeadOutlet>("head::after");

builder.Services.AddScoped(sp => new HttpClient { BaseAddress = new Uri(builder.HostEnvironment.BaseAddress) });
builder.Services.AddFluentUIComponents();
FluentWizard.LabelButtonDone = "完了";
FluentWizard.LabelButtonNext = "次へ";
FluentWizard.LabelButtonPrevious = "前へ";

await builder.Build().RunAsync();
```

```razor
@page "/"

<PageTitle>Home</PageTitle>

<FluentWizard>
    
    <Steps>
        <FluentWizardStep Label="Step1">
            何かの処理その1
        </FluentWizardStep>
        <FluentWizardStep Label="Step2">
            何かの処理その2
        </FluentWizardStep>
        <FluentWizardStep Label="Step3">
            何かの処理その3
        </FluentWizardStep>
    </Steps>
</FluentWizard>
```

**Comment by tomokusaba** [comment]
When I deployed this code to Static Web Apps, it gets garbled.

Home.razor

https://github.com/tomokusaba/FluentWizardSample

https://red-mushroom-0fcb8c710.4.azurestaticapps.net/

**Comment by vnbaaij** [comment]
Hi @tomokusaba,

Sorry, but all the text looks garbled to me as I can't read any of these characters...Can you point out where it is noot good?

**Comment by vnbaaij** [comment]
BTW I used a wrong issue number in commit [bd08636](https://github.com/microsoft/fluentui-blazor/commit/bd0863639a147d7d896942bfd51ad8f6e0cd95bc). I said this one but meant to link to #1295

**Comment by tomokusaba** [comment]
For some reason, it was due to the fact that Visual Studio had mistakenly saved it with Shift-JIS.
It displayed correctly in Visual Studio, displayed correctly locally, and for some reason I noticed that it was strange in Static Web Apps, but I didn't think about the character encoding problem.
Now that all your questions have been answered, we've updated our blog post to introduce FluentWizard to users in Japan.

Thank You.
